opentelemetry-sdk/src/opentelemetry/sdk/_metrics/export/__init__.py

```diff
@@ -77,3 +90,116 @@ def export(self, metrics: Sequence[Metric]) -> MetricExportResult:
 
     def shutdown(self) -> None:
         pass
+
+
+class PeriodicExportingMetricReader(MetricReader):
+    """`PeriodicExportingMetricReader` is an implementation of `MetricReader`
+    that collects metrics based on a user-configurable time interval, and passes the
+    metrics to the configured exporter.
+    """
+
+    def __init__(
+        self,
+        exporter: MetricExporter,
+        export_interval_millis: Optional[float] = None,
+        export_timeout_millis: Optional[float] = None,
+    ) -> None:
+        self._exporter = exporter
+        if export_interval_millis is None:
+            try:
```

Comment:
Not sure if we should check here, if a user makes a typo and sets this variable to something like `"10000p"` we would just set this to `60000` and inform nothing when the user wanted `10000`.

Reply:
Will add the warning log